src/openqdc/datasets/base.py

```diff
@@ -425,3 +490,48 @@ def __str__(self):
 
     def __repr__(self):
         return f"{self.__name__}"
+
+    @property
+    def _stats(self):
+        return self.__stats__
+
+    @property
+    def average_n_atoms(self):
+        """
+        Average number of atoms in a molecule in the dataset.
+        """
+        if self.__average_nb_atoms__ is None:
+            logger.info(PROPERTY_NOT_AVAILABLE_ERROR)
+            return 1
```

Comment:
Why 1? should be either an error or None

Reply:
It was a WIP patch to test different datasets on the multihead branch. Now we can just raise an error